libr/include/r_util/pj.h

```diff
@@ -37,6 +37,7 @@ R_API PJ *pj_knull(PJ *j, const char *k);
 R_API PJ *pj_kn(PJ *j, const char *k, ut64 n);
 R_API PJ *pj_kN(PJ *j, const char *k, st64 n);
 R_API PJ *pj_ks(PJ *j, const char *k, const char *v);
+R_API PJ *pj_ks_e(PJ *j, const char *k, const char *v, const char *e);
```

Comment:
i think it will be better to use pj_ke as name (basically because what we are encoding here is not really a string. it's raw data, and must be encoded somehow.

so better to use `e` and use const  ut8* and size_t len